index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta charset="ISO-8859-1">
+	<title>ROCA Observation Tool</title>
+	<link rel="stylesheet" href="obs.css">
+	<link rel="stylesheet" href="popupCSS.css">
+	<link rel="stylesheet" href="themify-icons.css">
+	<link rel="stylesheet" href="fonts/stylesheet.css">
+	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+	<script type="text/javascript" src="obs.js"></script>
+	<script type="text/javascript">
+	function GetURLParameter(sParam)
+	{
+   		var sPageURL = window.location.search.substring(1);
+    	var sURLVariables = sPageURL.split('&');
+    	for (var i = 0; i < sURLVariables.length; i++) 
+    	{
+        	var sParameterName = sURLVariables[i].split('=');
+        	if (sParameterName[0] == sParam) 
+        	{
+            	return sParameterName[1];
+        	}
+    	}
+	}
+	var classroomname=(GetURLParameter('classroomname'));
```

Comment:
If we separate building name and room number, maybe we can shorten the parameter names? Like "building" and "room"